### ISSUE TITLE
website layout enhancement

### ISSUE THREAD

**Issue by NOOB-3301** [SumitGorai01/TechieBlog#1]
Current website layout is only on single colour. 
![hyperss-2024-12-30_12-11-14](https://github.com/user-attachments/assets/b8c6b486-199b-4f0b-94be-ae9f4801bef4)

I will enhance it by adding responsiveness and make the design more eye catchy

**Comment by SumitGorai01** [comment]
@NOOB-3301 ,
Thank you for outlining these enhancements! I appreciate your suggestions .Adding responsiveness and enhancing the design will definitely make the site more visually appealing and user-friendly.
I can assign this issue to you. Please go ahead and start working on this.

Best regards,
Sumit

**Comment by prakhar123561** [comment]
Could I also contribute on the same issue ?

**Comment by NOOB-3301** [comment]
> Could I also contribute on the same issue ?

i am kinda done with this part, so after i make my pr , raise a different issue and contribute there

**Comment by NOOB-3301** [comment]
@SumitGorai01 i have made a pr . do check it out

**Comment by SumitGorai01** [comment]
@NOOB-3301 , 

Thanks for this PR #9 .

**Comment by NOOB-3301** [comment]
> @NOOB-3301 ,
> 
> Thanks for this PR #9 .

Do tell me if you merge or not . let me know if i have to make any changes

**Comment by SumitGorai01** [comment]
![image](https://github.com/user-attachments/assets/24ceb060-0701-4948-a4a4-ac67e310a46a)

after login , the latest post section have shown like this . please fix it to look better.

**Comment by LoneWolf4713** [comment]
hey I am willing to work on this, I will make use of a professional CSS Framework that will fix the UI, establish a good color scheme and make this website very professional looking.
here is a demo of my previous work: https://uni-match.netlify.com
Assign me.

**Comment by SumitGorai01** [comment]
@LoneWolf4713 ,
Thank you for outlining these enhancements! I appreciate your suggestions.
I can assign this issue to you. Please go ahead and start working on this.

Best regards,
Sumit,
Project Admin

**Comment by SumitGorai01** [comment]
@prakhar123561 ,

Thank you for outlining these enhancements! I appreciate your suggestions. You can also work on this issue but add some different section in website like image slider, faq section etc to look the website better.
I can assign this issue to you. Please go ahead and start working on this.

Best regards,
Sumit,
Project Admin

**Comment by LoneWolf4713** [comment]
hey @SumitGorai01 I am working on this, why did you assign @prakhar123561 . please keep one assignee as it would later lead to conflict to which one's PR might get merged

**Comment by SumitGorai01** [comment]
@LoneWolf4713 ,
You can work on this issue for now.  

And 

@prakhar123561 ,
Please wait until @LoneWolf4713  submits the PR.  
After that, you can work on improving the website layout or raise another issue to improve our website functionality.

**Comment by LoneWolf4713** [comment]
okay thanks.

**Comment by LoneWolf4713** [comment]
hey can I use Bulma CSS Framework instead of Tailwind for this?

**Comment by SumitGorai01** [comment]
@LoneWolf4713 ,
Yes, you can use Bulma CSS, but it should not affect the configuration of tailwind CSS that is used in the website. If you can use Bulma CSS along with tailwind css then there is no problem.

**Comment by prakhar123561** [comment]
Please assign me I will do your work.

**Comment by SumitGorai01** [comment]
@prakhar123561 ,
Kindly start working on this .
